db/migrate/20230627090845_create_like.rb

```diff
@@ -0,0 +1,10 @@
+class CreateLike < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :user, null: false, foreign_key: { to_table: :users }, index: true
```

Comment:
- [ ] Kindly change the references to `author` instead of `user` in the likes table as per the [diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png).